_overviews/scala3-book/control-structures.md

```diff
@@ -132,20 +145,57 @@ Otherwise you can write the code like this:
 for (i <- ints) println(i)
 ```
```

Comment:
I would suggest removing these lines (from “Otherwise you can write …”) since this is now covered by the “Scala 2” tab. What do you think?

Reply:
Sure, I was just trying to keep it as close to what it was before.

Reply:
Although if I remove it then users might think that they always need to write `do` ? :thinking:

Reply:
In Scala 3, that’s the syntax we want to recommend, yes.

Reply:
Fixed!